src/main/java/net/fabricmc/mappingio/format/enigma/EnigmaFileWriter.java

```diff
@@ -33,9 +33,6 @@ public boolean visitElementContent(MappedElementKind targetKind) throws IOExcept
 		} else if (targetKind == MappedElementKind.FIELD || targetKind == MappedElementKind.METHOD) {
 			writer.write(' ');
 			writer.write(desc);
-			writer.write('\n');
-		} else {
-			writer.write('\n');
```

Comment:
Intended?

Reply:
Yes, the line breaks are handled by `writePendingElementMetadata` now